galaxy.yml

```diff
@@ -79,6 +79,7 @@ tags:
   - katello
   - satellite
   - orcharhino
+  - infrastructure
```

Comment:
the full list of tags is:
- application
- cloud
- database
- infrastructure
- linux
- monitoring
- networking
- security
- storage
- tools
- windows

I think "infrastructure" fits best to Foreman and thus this collection.